e2e/test/PoolingOverAmqp.cs

```diff
@@ -6,10 +6,12 @@
 using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using static Microsoft.Azure.Devices.E2ETests.FaultInjectionPoolingOverAmqp;
 
 namespace Microsoft.Azure.Devices.E2ETests
 {
-    public static class PoolingOverAmqp
+    [TestClass]
+    public class PoolingOverAmqp
```

Comment:
Should these have a TestCategory?

Reply:
Definitely, I'll add iothub and e2e tags here